devops/docker/debian10-arm/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM multiarch/qemu-user-static:x86_64-arm as qemu
+FROM arm32v7/debian:buster
```

Comment:
same, use secure mirror `mcr.microsoft.com/mirror/docker/library/debian:buster`

Reply:
We should onboard the arm image - see https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/containers-bburns/microsoft-container-registry/microsoft-artifact-registry-also-known-as-microsoft-container-registry-docs/onboardingdocs/newimageonboarding